### ISSUE TITLE
Override mutation

### ISSUE THREAD

**Issue by wyllyan** [doug-martin/nestjs-query#926]
## 📚 Documentation

Hello! Does anyone have an example showing how to correctly override a mutation endpoint in the generated resolver? I can't find it in docs.

### Have you read the [Contributing Guidelines on issues](https://github.com/doug-martin/nestjs-query/blob/master/CONTRIBUTING.md)?

Yes

**Comment by smolinari** [comment]
Did you see this? https://doug-martin.github.io/nestjs-query/docs/graphql/resolvers#crudresolver

Scott

**Comment by wyllyan** [comment]
Yeah, but there is nothing about endpoints (createOne, etc) and its signature itself. I see the example with custom query, but I miss an example with mutation using MutationArgsType.

**Comment by smolinari** [comment]
I believe you can "turn off" the resolver in question then create your own via the resolver options. 

https://doug-martin.github.io/nestjs-query/docs/graphql/resolvers#disabling-endpoints

Not sure if that is the right thing to do or just override the method. 🤷 

Scott

**Comment by LuisAriasC** [comment]
I had some trouble with this but came up wit a silly solution.

As @smolinari said, you have to turn off the resolver via the resolver options and also in the CRUDResolver options like:

@Resolver(() => EntityDTO)
export class EntityResolver extends CRUDResolver(EntityDTO, {
  create: { one: { disabled: true } }, //OR ANY OTHER RESOLVER YOU NEED TO OVERRIDE
}) {
  constructor(readonly service: EntityService) {
    super(service);
  }
}
